app/jobs/repo_setup_job.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "concerns/repo_setup.rb"
+
+class RepoSetupJob < ApplicationJob
+  queue_as :repo_setup
+
+  def perform(assignment_repo_type, assignment_repo_id)
+    @assignment_repo = get_assignment_repo assignment_repo_type, assignment_repo_id
```

Comment:
Can we use parenthesis in the method call here? If you have different opinions let me know, but I find method calls with multiple arguments are kind of hard to follow with no parenthesis'